packages/talker_dio_logger/lib/dio_logs.dart

```diff
@@ -170,3 +176,16 @@ class DioErrorLog extends TalkerLog {
     return msg;
   }
 }
+
+///
+/// Get response time
+///
+int _getResponseTime(RequestOptions options) {
+  final triggerTime = options.extra[kDioLogsTimeStampKey];
+
+  if (triggerTime is int) {
+    return DateTime.now().millisecondsSinceEpoch - triggerTime;
+  }
+
+  return -1;
+}
```

Comment:
Please make the return value nullable integer (int?)
If triggerTime does not exist in the ```options.extra``` field, return null